ui/app/components/job-status/deployment-history.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+import Component from '@glimmer/component';
+import { watchRelationship } from 'nomad-ui/utils/properties/watch';
+import { inject as service } from '@ember/service';
+
+export default class JobStatusDeploymentHistoryComponent extends Component {
+  @service store;
+
+  async watchDeploymentHistory() {
+    const deployment = await this.args.deployment;
+    this.watch.perform(deployment, 10000);
```

Comment:
`question`:  This is going to be rendered on the `jobs.job.index` view which already has a `watchRelationship` on `latestDeployment`. Why are you performing this query here? Why isn't the template reactive to the updated entities in the Ember Data store? Looks like the `deployment` argument is being updated from the parent route and that's what's causing the update.